Assignment2/angellopezmonzoncillo.ttl

```diff
@@ -0,0 +1,14 @@
+@base <http://www.ontogrid.net/StickyNote/>
+@prefix class:<http://www.ontologies.org/ontologies/class#>
+@prefix person:<http://www.ontologies.org/ontologies/person#>
+@prefix sensor:<http://www.ontologies.org/ontologies/sensor#>
+@prefix computer:<http://www.ontologies.org/ontologies/computer#>
+@prefix measurement:<http://www.ontologies.org/ontologies/measurement#>
+		 
+:class01 class: include: sensor029;
+		 class: include: computer101;
+:sensor029 sensor: hasMeasurement: measurement8401;
+:computer101 computer: hasOwner: user10A;
+:measurement8401 measurement: hasTemperature: "29";

```

Comment:
the space between measurement and hasTemperature is wrong. Temperature RDF literal is not correctly typed.